terraform/modules/hub/hub_key_rotation.tf

```diff
@@ -116,6 +116,24 @@ resource "aws_security_group" "sg_hkr_lambda" {
   vpc_id = aws_vpc.hub.id
 }
 
+resource "aws_security_group_rule" "sg_hkr_lambda_ingress" {
```

Comment:
I dont think ingress is required is it?

Reply:
Ah is this ingress from the lambda?  If so, is it not just port 80?

Reply:
added incase it required any other resource need to talk to lambda since its attached to vpc

Reply:
its iingress for the lambda

Reply:
ah, in that case I think it should be removed unless we actually need it

Reply:
Done